migrations/201_workspaces__version__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspaces ADD COLUMN IF NOT EXISTS version bigint NOT NULL DEFAULT 1;
+ALTER TABLE workspace_logs ADD COLUMN IF NOT EXISTS workspace_version bigint NOT NULL;
```

Comment:
This will fail if we run it on production, because it won't know what to set `workspace_version` to for existing rows in `workspace_logs`. It either needs to allow NULL or it needs a DEFAULT.

Reply:
Changed to default=1